### DISCUSSION TITLE
GitHub Student Pack application details difficulties

### DISCUSSION THREAD

**Discussion by xeroxpoolikeactuallypoo** [community/community#57261] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm new to GitHub and i am a student who wants to apply for the GitHub Student developer pack.
Unfortunately everytime i get the same error:

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jun 5, 2023 for [@xeroxpoolikeactuallypoo](https://github.com/xeroxpoolikeactuallypoo).

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

I have a bio and my full name on my profile. Any fixes?

**Comment by samus-aran** [comment]
Hi @xeroxpoolikeactuallypoo 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this section would be key.
